apps/omg_performance/config/config.exs

```diff
@@ -2,4 +2,8 @@ use Mix.Config
 
 config :briefly, directory: ["/tmp/omisego"]
 
+config :byzantine_events,
```

Comment:
wouldn't we like to use the existing config entries for this?

Reply:
this?

Reply:
So this is the first test that touches Watcher's RPC (and also ch-ch RPC), hence I think it's better to have all the urls together (e.g. somebody would like to just run exit perftest)